server/odc-service/src/main/java/com/oceanbase/odc/service/quartz/QuartzJobService.java

```diff
@@ -59,11 +59,15 @@ public class QuartzJobService {
 
     @Autowired(required = false)
     @Qualifier(value = ("defaultScheduler"))
-    private Scheduler scheduler;
+    private Scheduler defaultScheduler;
+    @Autowired(required = false)
+    @Qualifier(value = ("commonScheduler"))
+    private Scheduler commonScheduler;
 
     public void changeQuartzJob(ChangeQuartJobParam req) {
 
         JobKey jobKey = QuartzKeyGenerator.generateJobKey(req.getJobName(), req.getJobGroup());
+        Scheduler scheduler = QuartzJob.class.isAssignableFrom(req.getJobClazz()) ? defaultScheduler : commonScheduler;
```

Comment:
clazz may be a typo?

Reply:
This is my spelling mistake, I have corrected it